product_docs/docs/pem/8/profiling_workloads/pem_sqlprofiler/installing_the_sql_profiler_plugin.mdx

```diff
@@ -114,6 +117,9 @@ When the installation is complete, the SQL Profiler plugin is ready to configure
 
 The SQL Profiler plugin isn't enabled when the installation process completes. This way you can restart the server at a convenient time. It also prevents the plugin from being loaded on systems where you don't need it all the time.
 
+!!! Note
+    If you connected to the PEM server with the PEM client, you must disconnect and reconnect with the server to enable SQL Profiler functionality. 
```

Comment:
Do you think this is needed:

If you connected to the PEM server with the PEM client before configuring SQL Profiler, you must disconnect and reconnect with the server to enable SQL Profiler functionality. 

I mean, adding "before configuring SQL Profiler" to the sentence will make it more clear.